questions/services/multiple_choice_handlers.py

```
@@ -0,0 +1,376 @@
+        ts = (
+            datetime.fromisoformat(datetime_str)
+            .replace(tzinfo=dt_timezone.utc)
+            .timestamp()
+        )
+        if new_options == current_options:  # no change
+            return
+        if len(new_options) == len(current_options):  # renaming
+            if any(v > 1 for v in Counter(new_options).values()):
+                ValidationError("new_options includes duplicate labels")
+        elif timezone.now().timestamp() < ts:
+            raise ValidationError("options cannot change during a grace period")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)